### ISSUE TITLE
SingleDropdownList and MultiDropdownList are not accessible (not 508 compliant)

### ISSUE THREAD

**Issue by allendol** [appbaseio/reactivesearch#1221]
Looks like **SingleDropdownList** and **MultiDropdownList** are changed to:
**<button type="button" role="button"**
even if I put id to the **<button id="123">** control and use **<label htmlFor={`123`}**, they are not accessible.
According to our 508 team, these until controls are converted to **select** control they are not accessible.
Any idea, Feedback?
